handlers/error.go

```diff
@@ -1,13 +1,25 @@
 package handlers
 
-import "net/http"
+import (
+	"log"
+	"net/http"
+)
 
 func errorHandler(w http.ResponseWriter, r *http.Request, status int) {
 	w.WriteHeader(status)
 
-	if status == http.StatusNotFound {
-		templates["message"].ExecuteTemplate(w, "layout", messageModel{"Error", "Page not found."})
-	} else {
-		templates["message"].ExecuteTemplate(w, "layout", messageModel{"Error", "An error has occurred."})
+	var message messageModel
+	switch status {
+	case http.StatusNotFound:
+		message = messageModel{"Error", "Page not found."}
+	case http.StatusServiceUnavailable:
+		message = messageModel{"Error", "Page not available at this time. Try again later."}
+	default:
+		message = messageModel{"Error", "An error has occurred."}
+	}
+
+	err := templates["message"].ExecuteTemplate(w, "layout", message)
+	if err != nil {
+		log.Println(err)
```

Comment:
How about changing this:
``` go
err := templates["message"].ExecuteTemplate(w, "layout", message)
if err != nil {
	log.Println(err)
```
to
``` go

if err := templates["message"].ExecuteTemplate(w, "layout", message); err != nil {
	log.Println(err)
```
?

Reply:
Line too long in my opinon.
But I have a solution:
- create renderTemplate() func
- call that instead of manually executing templates
- respects DRY
- can handle errors appropriately there